RELEASE-NOTES.txt

```diff
@@ -1,3 +1,18 @@
+1.5.0
+=====
+* Added a CLI tool that allows managing preview sites #1314
+* Dynamically determine sidebar breakpoint based on tab width #1196
+* Updated wordpress-components, Sentry, Electron, nokogiri and wp-babel makepot dependencies #1195, #1238, #1233, #1237, #1301
+* Fixed issue with opening logs on Windows #1245
+* Fixed and improved selecting WP version #1189
+* Added check if port is open before starting servers #1236, and #1281
+* Improved starting multiple sites with custom domains #1313
+* Filter out Automattic sites from the sync sites listing #1254 and #1262
```

Comment:
Noticed one minor thing:

```suggestion
* Filtered out Automattic sites from the sync sites listing #1254 and #1262
```

Reply:
Good catch! Applied the suggestion here https://github.com/Automattic/studio/pull/1324/files